skops/io/tests/test_external.py

```diff
@@ -46,6 +54,11 @@ def rank_data(clf_data):
     return X, y, group
 
 
+def _null(*args, **kwargs):
+    # used to prevent printing anything to stdout when calling visualize
```

Comment:
passing this as `sink` is disabling a fair amount of code, not just printing. Are you sure you want this?

Reply:
This is true. The relevant part of the code is executed, though, i.e. the tests would fail without the bugfix. If you want me to still change it, I can think of something (I guess auto fixture that overrides `sys.stdout`, not sure if it conflicts with pytest).

Reply:
yes, but you're also specifically testing this bug anyway. So it'd be nice to have `visualize` to actually test the whole thing, in case later we add something which would break things.

Reply:
Okay, makes sense, done.